content/docs/how-to-contribute.md

```diff
@@ -145,6 +145,13 @@ However, there are still some styles that the linter cannot pick up. If you are
 
 You may be interested in watching [this short video](https://www.youtube.com/watch?v=wUpPsEcGsg8) (26 mins) which gives an introduction on how to contribute to React.
 
+#### Available topics in this video directly linked:
+- build and test React locally [>](https://youtu.be/wUpPsEcGsg8?t=4m12s)
+- how to create and send pull requests to the React repository and how those are handled on project maintainer side [>](https://youtu.be/wUpPsEcGsg8?t=6m7s)
+- structure and organiztion of code: src files, entry points and module system [>](https://youtu.be/wUpPsEcGsg8?t=8m25s)
+- inspection and explanation of build/dist code pushed to npm registry [>](https://youtu.be/wUpPsEcGsg8?t=14m43s)
+- what to consider when adding new React features [>](https://youtu.be/wUpPsEcGsg8?t=19m15s)
```

Comment:
Cool idea!

How do you feel about showing the video timestamp as the link text rather than ">"? I think the ">" is a bit too small of a click target and also not immediately obviously. Maybe something more like this?

<img width="391" alt="screen shot 2017-11-07 at 1 35 28 pm" src="https://user-images.githubusercontent.com/29597/32496257-8a6d16de-c3c0-11e7-88ad-0da9b0137f12.png">

I also think these titles are a bit too long. I suggest shorter ones (like in the screenshot/mockup I attached).

Reply:
awesome, this one definitely makes much more sense and looks much better.